reinforcement_learning/bandits_recsys_movielens_testbed/bandits_movielens_testbed.ipynb

```diff
@@ -0,0 +1,749 @@
+{
```

Comment:
<p>local_mode default not set as True?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/949/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Yes, I missed this one, should be True.</p>